airbyte-integrations/connectors/destination-teradata/src/main/java/io/airbyte/integrations/destination/teradata/TeradataDestination.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.teradata;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.db.factory.DatabaseDriver;
+import io.airbyte.db.jdbc.JdbcUtils;
+import io.airbyte.integrations.BaseConnector;
+import io.airbyte.integrations.base.AirbyteMessageConsumer;
+import io.airbyte.integrations.base.Destination;
+import io.airbyte.integrations.base.IntegrationRunner;
+import io.airbyte.protocol.models.AirbyteConnectionStatus;
+import io.airbyte.protocol.models.AirbyteMessage;
+import io.airbyte.protocol.models.ConfiguredAirbyteCatalog;
+import java.util.function.Consumer;
+import io.airbyte.integrations.destination.jdbc.AbstractJdbcDestination;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Collections;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import io.airbyte.integrations.destination.ExtendedNameTransformer;
+
+public class TeradataDestination extends AbstractJdbcDestination  implements Destination {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(TeradataDestination.class);
+  public static final String DRIVER_CLASS = "com.teradata.jdbc.TeraDriver";
+  public static final String DATABASE_KEY = "database";
+  public static final String JDBC_URL_KEY = "jdbc_url";
+  public static final String JDBC_URL_PARAMS_KEY = "jdbc_url_params";
+  public static final String PASSWORD_KEY = "password";
+  public static final String USERNAME_KEY = "username";
+  public static final String SCHEMA_KEY = "schema";
+
+
+  public static void main(String[] args) throws Exception {
+    new IntegrationRunner(new TeradataDestination()).run(args);
+  }
+
+  public TeradataDestination() {
+    super(DRIVER_CLASS, new ExtendedNameTransformer(), new TeradataSqlOperations());
+  }
+
+  @Override
+  protected Map<String, String> getDefaultConnectionProperties(final JsonNode config) {   
+      return Collections.emptyMap();
+  }
+  @Override
+  public JsonNode toJdbcConfig(final JsonNode config) {
```

Comment:
Check how other connectors using JDBC do this function implementation.

Reply:
Took  destination-postgres as reference. Implemented as like this connector.